frontend/src/components/SearchBar.tsx

```diff
@@ -81,8 +115,34 @@ const SearchBar: React.FC<SearchBarProps> = ({ onSearch }) => {
         </div> */}
 
         {/* filter dropdowns for language */}
-        <div className='flex w-52 h-9 py-2 px-4 justify-between items-center shrink-0 border-solid border rounded-lg border-[#5F5F5F] bg-[#FFFDFD]'>
+        <div className='dropdown flex w-52 h-9 py-2 px-4 justify-between shrink-0 border-solid border rounded-lg border-[#5F5F5F] bg-[#FFFDFD]'>
```

Comment:
could be an issue on my end, but when I click on the chevron_down icon for the dropdowns, the dropdowns don't display the options (like Monday, Tuesday, etc). But when I hover over the chevron_down, the mouse icon does turn into the glove icon, so it's detecting something.